src/main/resources/pom-servers.xml

```diff
@@ -132,44 +132,32 @@
             <build>
                 <plugins>
                     <plugin>
-                        <groupId>net.wasdev.wlp.maven.plugins</groupId>
+                        <groupId>io.openliberty.tools</groupId>
```

Comment:
No jar created in the target directory, so the instructions in readme are no longer valid

>The generation of the executable jar file can be performed by issuing the following command
>
>    mvn clean package
>
>This will create an executable jar file **demo.jar** within the _target_ maven folder. This can be started by executing the following command
>
>   java -jar target/demo.jar

Reply:
The jar will be generated. I realised I made the type in my PR.

Reply:
Jar is now created but when running, I get this error 

      [ERROR   ] CWWKF0001E: A feature definition could not be found for microprofile-3.0

Reply:
I realised there is an defect in the plugin. The fix is in 3.1 plugin, which is available next week. Let's wait to get that version.